baseapp/baseapp.go

```diff
@@ -69,6 +69,8 @@ type BaseApp struct {
 
 	// flag for sealing
 	sealed bool
+	// Consensus parameters are set by (Request)InitChain and (Response)EndBlock
+	consensusParams *abci.ConsensusParams
```

Comment:
I used wrapped tmtypes.ConsensusParams so it is no more containing any pointers We still need to store params in the app however, because the states are wiped out at the commit while we need to preserve parameters